### PR TITLE
fix inconsistent limit check

### PR DESCRIPTION
**Pull request by EtienneLt**

### CODE REVIEW

File: src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java [hunk 1]
```diff
@@ -193,7 +193,8 @@ public void postRun(ShortCircuitRunContext runContext, AtomicReference<ReportNod
     protected void handleNonCancellationException(AbstractResultContext<ShortCircuitRunContext> resultContext, Exception exception, AtomicReference<ReportNode> rootReporter) {
         if (exception instanceof ShortCircuitException shortCircuitException && shortCircuitException.getType() == INCONSISTENT_VOLTAGE_LEVELS) {
             postRun(resultContext.getRunContext(), rootReporter, null);
-            sendResultMessage(resultContext, null);
+            notificationService.sendResultMessage(resultContext.getResultUuid(), resultContext.getRunContext().getReceiver(),
```
Comment (Mathieu-Deharbe) [inline src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java:196]: I don't get the point of this change. Does the HEADER_BUS_ID in `sendResultMessage` causes a problem ?

Comment (EtienneLt) [reply src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java:196]: in sendResultMessage result.getFaultResults().isEmpty is null so it throws an error

Comment (EtienneLt) [reply src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java:196]: I think it throws ShortCircuitException(MISSING_EXTENSION_DATA, "Missing short-circuit extension data");
but I'll check

Comment (EtienneLt) [reply src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java:196]: Ok the problem was the null argument, it was not managed in the method and it was throwing

Comment (Mathieu-Deharbe) [reply src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java]: Good. I prefer it that way.

**Review by Mathieu-Deharbe — APPROVED** [review]

**Pull request merged**